Sources/MockoloFramework/Models/ParsedEntity.swift

```diff
@@ -131,14 +125,31 @@ struct AnnotationMetadata {
     var combineTypes: [String: CombineType]?
 }
 
+struct GenerationArguments {
```

Comment:
[imho]

```suggestion
struct RenderArguments {
```

I might prefer RenderArguments to align word choice with RenderContext, though it is not must.

Reply:
The `GenerationArguments` means the arguments of `func generate`.
The value is unique (fixed) for applying `generate`, unlike the render process.

Reply:
I could not distinguish the difference. Thank you for your information.🙇‍♂️